docs/api.md

```diff
@@ -496,6 +496,93 @@ await webcontainerInstance.fs.writeFile(
 );
 ```
 
+### ▸ `watch`
+
+Watch for changes to a given file or directory.
+
+<h4 id="watch-signature">
+  <a id="watch-signature">Signature</a>
+  <a href="#watch-signature" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+`watch(path: string, options: Options, Callback): Watcher`
+
+`watch(path: string, Callback): Watcher`
+
+<h4 id="watch-options">
+  <a id="watch-options"><code>Options</code></a>
+  <a href="#watch-options" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+```ts
+interface Options {
+  encoding?: BufferEncoding | null;
```

Comment:
What is the meaning of providing `null` for `encoding`? If there isn't any we should probably just type it as:

```ts
interface Options {
  encoding?: BufferEncoding;
  ...
}
```

Reply:
BufferEncoding is defined at the bottom of the file

Reply:
`encoding?: BufferEncoding | null` appears to be used elsewhere in the API